ClassModules/Classes/DruidClasses.lua

```diff
@@ -220,6 +221,11 @@ function TRB.Classes.Druid.BalanceSpells:New()
     self.touchTheCosmos = TRB.Classes.SpellBase:New({
         id = 450360
     })
+    self.starlord = TRB.Classes.SpellBase:New({
+        id = 202345,
+        isTalent = true,
+        buffId = 279709
+    })
```

Comment:
Since you're actually using the `buffId` to track values, it makes more sense to have that be the base `id` and instead set the current `id` to be `talentId`. This also cleans up/futureproofs other parts of the code as I continue to split and specify responsibilities. 

```suggestion
    self.starlord = TRB.Classes.SpellBase:New({
        id = 279709,
        talentId = 202345,
        isTalent = true
    })
```

Then in `ClassModules/Druid.lua` you can change the part I've suggested to not use `buffId`.